web/app/js/components/TopModule.jsx

```diff
@@ -237,7 +239,8 @@ class TopModule extends React.Component {
     }
 
     if (d.base.proxyDirection === "INBOUND") {
-      this.props.updateNeighbors(d.requestInit.source, _get(d, "requestInit.sourceMeta.labels"), null);
+      let unmeshedSources = this.props.updateNeighbors(d.requestInit.source, _get(d, "requestInit.sourceMeta.labels"), null);
+      topResults[eventKey].meshed = !_has(unmeshedSources, d.base.source.owner);
```

Comment:
this check doesn't quite work when the resource we're doing the Top on is a `pod`. E.g. with web meshed, if you go to the Web's Resource Detail page for pod (e.g. http://localhost:8084/namespaces/emojivoto/pods/web-6b4675554d-ldtzm) you'll see that the tap icon continues to appear for all rows.

I think you'd need something like:


```suggestion
      let isPod = this.props.query.resource.split("/")[0] === "pod";
      if (isPod) {
        topResults[eventKey].meshed = !_has(unmeshedSources, "pod/" + d.base.source.pod);
      } else {
        topResults[eventKey].meshed = !_has(unmeshedSources, d.base.source.owner);
      }
```

Reply:
Ok, understood :+1: